### PR TITLE
Adapt Starlette doc usage according starlette v13

### PR DESCRIPTION
**Pull request by petrus-v**
As far I understand, [route decorator is on the way to be deprecated](https://github.com/encode/starlette/pull/704).

I have seen that starlette is freezed at ``starlette>=0.12,<0.13`` hope this helps on the way to move ahead !

### CODE REVIEW

**Comment by florimondmanca** [comment]
@petrus-v Is `tartiflette-asgi` working for you with Starlette 0.13? I notice we still have #100 open - I initially thought there were some incompatibilities due to the way Starlette does lifespan in 0.13.

**Comment by petrus-v** [comment]
> @petrus-v Is `tartiflette-asgi` working for you with Starlette 0.13? I notice we still have #100 open - I initially thought there were some incompatibilities due to the way Starlette does lifespan in 0.13.

Sorry I've create #106 just before reading issues where I've notice your reply...

So yes, this example is working with both starlette versions: 0.12.3 and using #106 with v0.13.3.

Thanks for pointing me `Mount` class and I'll also change example code as suggested.

**Comment by petrus-v** [comment]
According last change using `Mount` I wonder if we should change previous chapter ?

In an other hand I've notice that ``app.add_event_handler("startup", graphql.startup)`` is still required using #106 I'll investigate If I could improve that next days.

**Comment by petrus-v** [comment]
> According last change using `Mount` I wonder if we should change previous chapter ?
> 
> In an other hand I've notice that `app.add_event_handler("startup", graphql.startup)` is still required using #106 I'll investigate If I could improve that next days.

Last commit looks better using on_startup param from starlette() app (availiable from starlette v0.13).

File: docs/usage.md [hunk 1]
```diff
@@ -191,10 +183,18 @@ async def hello(parent, args, context, info):
...
+
+# Create a Starlette application, mounting the 'TartifletteApp' instance.
 
-app.mount("/graphql", graphql)
-app.add_event_handler("startup", graphql.startup)
+routes = [
+    Route("/", endpoint=home),
+    Mount("/graphql", graphql),
+]
+app = Starlette(routes=routes, on_startup=graphql.startup)
```
Comment (petrus-v) [inline docs/usage.md:197]:
```suggestion
app = Starlette(routes=routes, on_startup=[graphql.startup])
```

File: docs/usage.md [hunk 2]
```diff
@@ -127,7 +127,7 @@ In general, you'll need to do the following:
 1. Register the startup lifespan event handler on the main ASGI application. (Frameworks typically expose a method such as `.add_event_handler()` for this purpose.)
 
 !!! important
-    The startup event handler is responsible for preparing the GraphQL engine (a.k.a. [cooking the engine](https://tartiflette.io/docs/api/engine#cook-your-tartiflette)), e.g. loading modules, SDL files, etc.
+The startup event handler is responsible for preparing the GraphQL engine (a.k.a. [cooking the engine](https://tartiflette.io/docs/api/engine#cook-your-tartiflette)), e.g. loading modules, SDL files, etc.
```
Comment (florimondmanca) [inline docs/usage.md:130]: This tab is actually important for rendering - probably due to autoformatting in your editor, but we should keep it in. :-)

```suggestion
    The startup event handler is responsible for preparing the GraphQL engine (a.k.a. [cooking the engine](https://tartiflette.io/docs/api/engine#cook-your-tartiflette)), e.g. loading modules, SDL files, etc.
```

**Comment by florimondmanca** [comment]
Unrelated note, but it looks like Travis CI is not triggering GitHub checks anymore, not sure why that is. Looks like I shall port this repo to Azure Pipelines too…

File: docs/usage.md (hunk 2 above)
Comment (petrus-v) [reply docs/usage.md]: Indeed autoformat... sorry for the noise

**Comment by petrus-v** [comment]
I'm closing this PR, those changes have been integrated into #106

**Pull request closed without merging**